lib/rubocop/cop/rspec/rails/minitest_assertions.rb

```diff
@@ -28,11 +30,14 @@ class MinitestAssertions < Base
           RESTRICT_ON_SEND = %i[
             assert_equal
             assert_not_equal
+            assert_includes
+            assert_not_includes
             assert_nil
             assert_not_nil
```

Comment:
There’s an “a-ha” moment, since all those assert_no_* are from test-unit, not minitest.

But I’d say we don’t care much if there’s a corresponding native RSpec expectation/matcher we can suggest using instead.

test-unit assertions https://www.rubydoc.info/gems/test-unit/2.3.2/Test/Unit/Assertions
Minitest assertions https://ruby-doc.org/stdlib-3.0.0/libdoc/minitest/rdoc/Minitest/Assertions.html

Reply:
Yea though also they're defined by Rails which is actually why I was including them: https://github.com/rails/rails/blob/main/activesupport/lib/active_support/test_case.rb#L206